demos/janssen-tarp/browser-extension/src/features/authentication/components/AuthFlowInputs.tsx

```
@@ -121,12 +121,81 @@ export default function AuthFlowInputs({
+            }
+          });
+
+        } else {
+          // ── Normal flow ──────────────────────────────────────────────────────
+          chrome.identity.launchWebAuthFlow({ url: authzUrl, interactive: true }, (responseUrl) => {
+            if (chrome.runtime.lastError || !responseUrl)
+              reject(new Error(chrome.runtime.lastError?.message || 'No redirect URL'));
+            else resolve(responseUrl);
+          });
+        }
       });
```

Biome (2.4.14)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)